File: Day-13-Abstract Classes/day13 .php
class MyBook extends Book{

    private $price;

    function __construct($title,$author,$price){
        $this->title = $title;
        $this->author = $author;
        $this->price = $price;
    }


     function display(){
        echo "Title: ".$this->title;
        echo "Author: ".$this->author;
        echo "Price: ".$this->price;
    }

}

File: Day-13-Abstract Classes/day13.cs
class MyBook : Book
{
  private int price = 0;
  public MyBook(String title, String author, int price):base(title, author)
  {
    this.price = price;
  }

  public override void display()
  {
          Console.WriteLine("Title: "+ title);
          Console.WriteLine("Author: " + author);
          Console.WriteLine("Price: " + price);
  }
}
     
